diff --git a/node_modules/remix-theme/dist/index.d.ts b/node_modules/remix-theme/dist/index.d.ts
index 9b33c46..f547e4e 100644
--- a/node_modules/remix-theme/dist/index.d.ts
+++ b/node_modules/remix-theme/dist/index.d.ts
@@ -32,6 +32,8 @@ export interface ThemeProviderProps {
     attribute?: string | 'class';
     /** Mapping of theme name to HTML attribute value. Object where key is the theme name and value is the attribute value */
     value?: ValueObject;
+    /** Children */
+    children: React.ReactNode;
 }
 export declare const useTheme: () => UseThemeProps;
 interface ValueObject {
